editions/dev/tiddlers/Cascade Mechanism.tid

```diff
@@ -0,0 +1,83 @@
+created: 20240802065815656
+modified: 20240802065836064
+title: Cascade Mechanism
+type: text/vnd.tiddlywiki
+
+Basic usage is already explained in the official documentation. This guide will tell you how to add new cascade to the ~TiddlyWiki core or your own plugins.
+
+!! Add new cascade transclusion
+
+!!! The default template as a fallback
+
+(omitted here, see the latest version on [ext[Github|https://github.com/TiddlyWiki/TiddlyWiki5/blob/master/core/ui/ViewTemplate/tags/default.tid]])
```

Comment:
It would be better to be self contained and transclude the shadow tiddler as a subtiddler from the core plugin. This will ensure that the real shadow is displayed even if it has been overridden.

Reply:
This requires /dev doc site use prerelease of TW, will it be?

Reply:
Or we merge this into master branch.

Reply:
Hi @linonetwo yes it looks like this will need to be merged to "master"